apps/chef_test/src/chef_test_suite_helper.erl

```diff
@@ -106,8 +127,90 @@ make_az_id(Prefix) ->
 actor_id() ->
     make_az_id(<<"ffff">>).
 
+make_orgs() ->
+    make_org(),
+    make_other_org(),
+    OrgConfig = org_config(),
+    confirm_org_setup(OrgConfig),
+    OrgConfig.
+
+make_org() ->
+    Org = chef_object:new_record(oc_chef_organization, nil, ?ORG_AUTHZ_ID,
+                                 {[{<<"name">>, ?ORG_NAME}, {<<"full_name">>, ?ORG_NAME}]}),
+    ok = chef_db:create(Org, context(), ?ORG_AUTHZ_ID).
+
+make_other_org() ->
+    Org = chef_object:new_record(oc_chef_organization, nil, ?OTHER_ORG_AUTHZ_ID,
+                                 {[{<<"name">>, ?OTHER_ORG_NAME}, {<<"full_name">>, ?OTHER_ORG_NAME}]}),
+    ok = chef_db:create(Org, context(), ?OTHER_ORG_AUTHZ_ID).
+
+org_config() ->
+    OrgId = chef_db:fetch_org_id(context(), ?ORG_NAME),
+    OtherOrgID = chef_db:fetch_org_id(context(), ?OTHER_ORG_NAME),
+    ct:pal("Other org: ~p", [OtherOrgID]),
+    [{org_id, OrgId}, {other_org_id, OtherOrgID}].
+
+confirm_org_setup(Config) ->
+    ExpectedOrgId = proplists:get_value(org_id, Config),
+    ExpectedOtherOrgId = proplists:get_value(other_org_id, Config),
+
+    ct:pal("ORG: ~p~n", [ExpectedOrgId]),
```

Comment:
a reminder to nuke the output here if we don't need it after testing/debugging is done